src/app/report-rollup/report-summary-graphs/report-summary-graphs.component.ts

```diff
@@ -59,14 +59,13 @@ export class ReportSummaryGraphsComponent implements OnInit {
     });
   }
 
-  ngOnDestroy() {
-    this.reportSummaryGraphService.clearData();
```

Comment:
Why was this removed?

Reply:
Because that method no longer needs to be executed due to the updates done to the configuration of the reportSummaryGraphService